### PR TITLE
Added functionality for unit testing Room class getter and setters.

### PR DESCRIPTION
**Pull request by adamarthurcone**
Functionality for unit testing Room class getter and setters did not exist previously.   A total of four unit tests were added to a new file called RoomTest.java.
Resolves: #339

### CODE REVIEW

File: test/edu/wright/cs/raiderplanner/model/RoomTest.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+package edu.wright.cs.raiderplanner.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Created by Adam Cone 09/22/2018
```
Comment (rsanchez-wsu) [inline test/edu/wright/cs/raiderplanner/model/RoomTest.java]: First sentence in Javadoc needs a period, or CheckStyle complains.

File: test/edu/wright/cs/raiderplanner/model/RoomTest.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+	 * @throws Exception to handle when any of the required objects cannot be instantiated.
+	 */
+	@BeforeEach
+	public void setUp() throws Exception {
+		bd1 = new Building("CEG", 0.0, 0.0);
+		bd2 = new Building("MTH", 0.0, 0.0);
+		rm1 = new Room("321", bd1);
+		rm2 = new Room("123", bd2);
+	}
+
```
Comment (rsanchez-wsu) [inline test/edu/wright/cs/raiderplanner/model/RoomTest.java]: Please remove trailing whitespace.

File: test/edu/wright/cs/raiderplanner/model/RoomTest.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+	
+	/**
+	 * This test case attempts to retrieve building object information.
+	 * @throws Exception to handle when any of the required objects cannot be instantiated.
+	 */
+	@Test
+	public void getBuilding() throws Exception {
+		assertEquals(bd1, rm1.getBuilding());
+	}
+
```
Comment (rsanchez-wsu) [inline test/edu/wright/cs/raiderplanner/model/RoomTest.java]: Please remove trailing whitespace.

File: test/edu/wright/cs/raiderplanner/model/RoomTest.java [hunk 4]
```diff
@@ -0,0 +1,99 @@
+/*
```
Comment (rsanchez-wsu) [inline test/edu/wright/cs/raiderplanner/model/RoomTest.java]: This header comment is not properly formatted.  Please have a look at other examples from the project and match those.  Make sure to include your name.

**Comment by adamarthurcone** [comment]
I have made the changes that you requested.

File: test/edu/wright/cs/raiderplanner/model/RoomTest.java (hunk 4 above)
Comment (adamarthurcone) [reply test/edu/wright/cs/raiderplanner/model/RoomTest.java]: I believe this is corrected now.

**Pull request merged**